README.md

```diff
@@ -43,17 +43,17 @@ conversion tools to produce inference-optimized programs.
 * See [Direct Quantization with SHARK Tank](./docs/quantization.md)
   for information about quantization support.
 
-### [`tuner/`](./tuner/)
+### [`sharktuner/`](./sharktuner/)
 
-[![CI - Tuner](https://github.com/nod-ai/shark-ai/actions/workflows/ci-tuner.yml/badge.svg?event=push)](https://github.com/nod-ai/shark-ai/actions/workflows/ci-tuner.yml?query=event%3Apush)
+[![CI - sharktuner](https://github.com/nod-ai/shark-ai/actions/workflows/ci-tuner.yml/badge.svg?event=push)](https://github.com/nod-ai/shark-ai/actions/workflows/ci-tuner.yml?query=event%3Apush)
```

Comment:
This badge is broken. Rename `ci-tuner.yml` to `ci-sharktuner.yml` here as well?

* ![image](https://github.com/user-attachments/assets/0b6e3d59-5790-49ed-ba0f-7b82705e5e69)
* ![image](https://github.com/user-attachments/assets/754b8864-13fd-4321-ac16-a0c891c68486)